documentation/domains/Cluster.md

```diff
@@ -23,7 +23,7 @@ The specification of the operation of the WebLogic cluster. Required.
 | `maxUnavailable` | integer | The maximum number of cluster members that can be temporarily unavailable. Defaults to `domain.spec.maxClusterUnavailable`, which defaults to 1. |
 | `replicas` | integer | The number of cluster member Managed Server instances to start for this WebLogic cluster. The operator will sort cluster member Managed Server names from the WebLogic domain configuration by normalizing any numbers in the Managed Server name and then sorting alphabetically. This is done so that server names such as "managed-server10" come after "managed-server9". The operator will then start Managed Server instances from the sorted list, up to the `replicas` count, unless specific Managed Servers are specified as starting in their entry under the `managedServers` field. In that case, the specified Managed Server instances will be started and then additional cluster members will be started, up to the `replicas` count, by finding further cluster members in the sorted list that are not already started. If cluster members are started because of their related entries under `managedServers`, then this cluster may have more cluster members running than its `replicas` count. Defaults to `domain.spec.replicas`, which defaults 1. |
 | `restartVersion` | string | Changes to this field cause the operator to restart WebLogic Server instances. More info: https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#restarting-servers. |
-| `serverPod` | [Server Pod](#server-pod) | Customization affecting the generation of Pods for WebLogic Server instances. |
```

Comment:
I don't think that cluster level 'serverPod' tuning has an effect a domain level introspector. The wording could be "Customization affecting the generation of Pods for the cluster's WebLogic Server instances."

Reply:
The `serverPod` is shared between domain, cluster and server scopes. I can't give different documentation for different scopes. I'll start a slack thread to discuss this.